contribution/lang/fr.json

```diff
@@ -348,154 +348,154 @@
  	 	 	 	"eng": "inject it to cure any(*conditions apply) injury!"
  	 	 	}
  	 	},
  	 	"healing5": {
  	 	 	"name": {
- 	 	 	 	"trans": "Military Nano Medic Bot",
+ 	 	 	 	"trans": "Nano Bot Médical Militaire",
  	 	 	 	"eng": "Military Nano Medic Bot"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "Injectable nano machines that are standard issue for military use, capable of patching injuries while in combat.",
+ 	 	 	 	"trans": "Nano machines injectables standards pour un usage militaire, soins possibles pendant le combat.",
  	 	 	 	"eng": "Injectable nano machines that are standard issue for military use, capable of patching injuries while in combat."
  	 	 	}
  	 	},
  	 	"healing6": {
  	 	 	"name": {
- 	 	 	 	"trans": "Advanced Nano Medic Bot",
+ 	 	 	 	"trans": "Nano Bot Médical Avancé",
  	 	 	 	"eng": "Advanced Nano Medic Bot"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "Injectable nano machines that are used in the military. They are capable of patching injuries while in combat.",
+ 	 	 	 	"trans": "Nano machines injectables utilisées dans l'armée. Elles peuvent soigner les blessures pendant le combat.",
  	 	 	 	"eng": "Injectable nano machines that are used in the military. They are capable of patching injuries while in combat."
  	 	 	}
  	 	},
  	 	"healing7": {
  	 	 	"name": {
- 	 	 	 	"trans": "High Quality Nano Medic Bot",
+ 	 	 	 	"trans": "Nano Bot Médical de Haute Qualité",
  	 	 	 	"eng": "High Quality Nano Medic Bot"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "injectable nano machines that is for advance military use, capable of patching injury in combat situation",
+ 	 	 	 	"trans": "Nano machines injectables pour un usage militaire avancé, soins possibles dans des situations de combat",
  	 	 	 	"eng": "injectable nano machines that is for advance military use, capable of patching injury in combat situation"
  	 	 	}
  	 	},
  	 	"healing8": {
  	 	 	"name": {
- 	 	 	 	"trans": "Rare Nano Medic Bot",
+ 	 	 	 	"trans": "Nano Bot Médical Rare",
  	 	 	 	"eng": "Rare Nano Medic Bot"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "injectable nano machines that is for advance military use, capable of patching injury in combat situation",
+ 	 	 	 	"trans": "Nano machines injectables pour un usage militaire avancé, soins possibles dans des situations de combat",
  	 	 	 	"eng": "injectable nano machines that is for advance military use, capable of patching injury in combat situation"
  	 	 	}
  	 	},
  	 	"healingArea1": {
  	 	 	"name": {
- 	 	 	 	"trans": "Pain Away Spray",
+ 	 	 	 	"trans": "Spray Anti-douleurs",
  	 	 	 	"eng": "Pain Away Spray"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "Do you and your friend have headache together? use Pain Away Spray™ to relief your pain!",
+ 	 	 	 	"trans": "Avez-vous et votre ami des maux de tête en commun ? Utilisez un Spray Anti-douleurs™ pour vous soulager !",
```

Comment:
Est-ce que vous et votre ami avez des maux de tête en commun ? ...

Reply:
I guess we could even translate without "en commun", like this: "Votre ami et vous avez des mots de tête ?". putting the friend first is more French than the original, as in the French language, we place others before ourselves in a group.
thus, saying "votre ami et vous" is better than "avez-vous et votre ami" [...]

Reply:
i dont agree on that one, i really think it should be the form i suggested, because it sounds more popular. Those 2 other forms are too formal for a cyberpunk game

Reply:
so more popular but wrong is ok for you? -_-'
it's because we use improper and wrong French forms and sentences that our language is becoming poorer and dirtier.

Reply:
I'll accept your suggestion in order to complete the merge, but I strongly disagree in making my language poorer